app/services/piv_cac_service.rb

```diff
@@ -0,0 +1,51 @@
+require 'net/https'
+
+module PivCacService
+  class << self
+    def decode_token(token)
+      token_present(token) &&
+        token_decoded(token)
+    end
+
+    def piv_cac_service_link
+      Figaro.env.piv_cac_service_url!
```

Comment:
What do you think about requiring these in `config/initializers/figaro.rb`, so that if they are missing, an error will be raised during app initialization, as opposed to during runtime once this method is called?

Reply:
I think I'll remove the bang for now since we aren't enabling this in production. But once we're close to putting it into production, definitely. We also don't know what those values will be in any environments yet.